questions.ts

```diff
@@ -103,6 +103,7 @@ const getVectorDbChoices = (framework: TemplateFramework) => {
     { title: "Astra", value: "astra" },
     { title: "Qdrant", value: "qdrant" },
     { title: "ChromaDB", value: "chroma" },
+    { title: "Weaviate", value: "weaviate" },
```

Comment:
only show for framework === "fastapi"

Reply:
i removed the weaviate code for TS so it no longer shows for TS. See this availableChoices: https://github.com/run-llama/create-llama/blob/878cfc2ca1737cee25df897bb6c66574868d10cb/questions.ts#L112

Reply:
ah cool, did totally forget that we have this :D